### ISSUE TITLE
New line is added between assignment and comment

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#24]
A new line is added between an assignment and a comment in the following example:

```
// in
a := 2;
//comment

// out
a := 2;

//comment

// expected
a := 2;
//comment
```

See also the gif in #23.

- STweep v3.0.0-beta
- [settings.txt](https://github.com/Barteling/STweep/files/5988997/settings.txt)

**Comment by Barteling** [comment]
Seems to happen with when Blank lines at file end  > 1 
bug #23 is probably related.

**Comment by Barteling** [comment]
Fixed in 3.0.1-Beta
